package.json

```diff
@@ -15,13 +15,13 @@
   "jsdelivr": "dist-browser/geotiff.js",
   "exports": {
     ".": {
-      "import": "./src/geotiff.js",
+      "import": "./dist-module/geotiff.js",
       "require": "./dist-node/geotiff.js",
       "browser": "./dist-browser/geotiff.js"
     }
   },
   "files": [
-    "src",
+    "dist-module",
```

Comment:
I was not aware of the impact of the `"files"` option in #269. Sorry, I should have checked that.

Reply:
Nevermind. Thanks for your contribution!